### DISCUSSION TITLE
Is there any option to install Copilot4Eclipse using "install new software" option instead of Eclipse marketplace?

### DISCUSSION THREAD

**Discussion by lathapatil** [Genuitec/Copilot4Eclipse-community#13] [Help] [answered]
Eclipse marketplace is somehow disabled in my organization for various reasons. Is there a option or update-site available for Copilot4Eclipse to install in Eclipse via "Install new software ?"

**Comment by wayneparrott** [comment, marked as answer]
Hi, 

For corporate level installations, i.e., non-Marketplace installations, we provide a corporate licensing option for Copilot4Eclipse. To determine if this what your need please see the following FAQ topic: : https://www.genuitec.com/products/copilot4eclipse/docs/faq#corporate-license

W

**Reply by lathapatil** [reply]
Thanks for the quick response. The [corporate licensing options](https://www.genuitec.com/products/copilot4eclipse/docs/faq/faq#licensing) is not accessible for me

**Reply by toddewilliams** [reply]
Sorry for the typo in the link. Please see the Licensing section of the FAQ for Copilot4Eclipse [here](https://www.genuitec.com/products/copilot4eclipse/docs/faq#licensing). We look forward to hearing from you.

**Reply by ghost** [reply]
Is there a plan to have Inline chat session in Eclipse editors ? and #terminalSelection /fix

**Reply by wayneparrott** [reply]
@Latha-Patil we are definitely working to provide deeper integration of Copilot4Eclipse features across the Eclipse IDE. We will be sharing more details soon.
